@xen-orchestra/defined/.USAGE.md

```diff
@@ -0,0 +1,21 @@
+The defined() function returns the first non-undefined value from a list of arguments, evaluating functions if needed.
+If only undefined values, return undefined.
+
+```js
+import defined from '@xen-orchestra/defined/index.js'
+
+defined(undefined, 'foo', 42)
+// Returns 'foo'
+
+defined([undefined, null, 10])
+// Returns null
+
+defined([undefined, undefined], [undefined, undefined, 10])
+// Returns 10
+
+defined(() => 'bar', 42)
+// Returns 'bar'
+
+defined(undefined, undefined)
+// Returns undefined
+```
```

Comment:
Good initiative to document this :slightly_smiling_face: 

Move this in a separate commit.